app/constantData/demoData.ts

```diff
@@ -0,0 +1,17 @@
+export const courseDetails = {
+  id: 1,
+  teacherName: "Mr. John Doe",
+  Section: "A",
+  value: 235,
+  bookList: ["Book 01", "Book 02", "Book 03"],
+  authorList: ["Author 01", "Author 02", "Author 03"],
+};
+
+function generateCourseList(totalItems: number) {
+  return Array.from({ length: totalItems }, (_, index) => ({
```

Comment:
we can use an implicit return.
```suggestion
  Array.from({ length: totalItems }, (_, index) => ({
```

Reply:
done